### DISCUSSION TITLE
Approved but GitHub Education benefits not provide

### DISCUSSION THREAD

**Discussion by jihaanjj** [community/community#135736] [GitHub Education] [answered]
### Hi everyone,

I got approved on August 9, but I still have not received my benefits yet. 

![Screenshot 2024-08-14 113558](https://github.com/user-attachments/assets/63f3a89a-903f-4878-877a-c9bd8722a189)

**Comment by SHresTho12** [comment]
It takes some days sometimes. Mine took 4 days . Wait for the email.

**Comment by davevad93** [comment, marked as answer]
Hi @JejeTrue ,

When the status label is purple and states “***your benefits are now available***” it means you should have access to them.

Go to your [education page](https://education.github.com/learner/learn), scroll down a bit to the `Student Developer Pack offers` section, click `+` to browse all the offers and select the ones you are interested in, if you can access them correctly it means you have full access to your student benefits.

![000000](https://github.com/user-attachments/assets/33aca3c6-4ade-4b23-b2e7-93b311324db5)

If you're unable to access any of the offers, you should contact Education support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue.

EDIT: The pack offers page is returning a 404 error, the education platform may be experiencing issues today and that may be why you haven't been able to access your benefits yet, try waiting a while until it works again.

**Reply by davevad93** [reply]
However, the link for the `Student Developer Pack` offers is working, check it out [here](https://education.github.com/pack), you can explore the available offers and access your benefits.

**Reply by jihaanjj** [reply]
thank you, problem solved

**Comment by nanthu123tech** [comment]
It is not getting process for an educational benefits the application is denied without any reason
